node/grpc/server.go

```diff
@@ -251,7 +253,77 @@ func (s *Server) StoreChunks(ctx context.Context, in *pb.StoreChunksRequest) (*p
 }
 
 func (s *Server) StoreBlobs(ctx context.Context, in *pb.StoreBlobsRequest) (*pb.StoreBlobsReply, error) {
-	return nil, errors.New("StoreBlobs is not implemented")
+	start := time.Now()
+
+	// Validate the request
+	if in.GetReferenceBlockNumber() == 0 {
```

Comment:
Should we put some of this in a validation function?

Reply:
Can we generalize the `validateStoreChunksRequest` and reuse in both places?

Reply:
If this method is getting too long, I'd prefer creating another method `validateStoreBlobsRequest` instead of generalizing `validateStoreChunksRequest` and using it in both places. 
After the migration, StoreChunks method will be deprecated. I think it's better to make two codepaths somewhat independent and clear for each use case, so that eventually when one codepath is removed, what's left is clear for those without context.